heron/examples/src/java/com/twitter/heron/examples/CustomGroupingTopology.java

```diff
@@ -18,17 +18,17 @@
 import java.util.List;
 import java.util.Map;
 
-import backtype.storm.Config;
-import backtype.storm.StormSubmitter;
-import backtype.storm.generated.GlobalStreamId;
-import backtype.storm.grouping.CustomStreamGrouping;
-import backtype.storm.task.OutputCollector;
-import backtype.storm.task.TopologyContext;
-import backtype.storm.task.WorkerTopologyContext;
-import backtype.storm.topology.OutputFieldsDeclarer;
-import backtype.storm.topology.TopologyBuilder;
-import backtype.storm.topology.base.BaseRichBolt;
-import backtype.storm.tuple.Tuple;
+import org.apache.storm.Config;
```

Comment:
This change, along with the introduction of `GlobalMetrics`, is here because I changed to only include "org/apache/**" in storm lib. See: https://github.com/twitter/heron/pull/1811/files#diff-cf85344880ea3bb51b03100c97fb7a82R22

This means I have to drop any use of backtype in examples. This is related to https://github.com/twitter/heron/pull/1811#discussion_r112565244